packages/nc-gui/composables/useProject.ts

```diff
@@ -169,6 +174,10 @@ export const useProject = createSharedComposable(() => {
     $e('c:themes:change')
   }
 
+  async function hasEmptyOrNullFilters() {
+    return await api.project.hasEmptyOrNullFilters(projectId.value)
```

Comment:
`await` can be removed

Reply:
May I know why? `hasEmptyOrNullFilters` in `projectApis.ts` is async.

Reply:
both will have the same effect, it can be simplified to

```
function hasEmptyOrNullFilters() {
    return api.project.hasEmptyOrNullFilters(projectId.value)
    
}
```